app/customerstatusmanager.py

```diff
@@ -1,8 +1,22 @@
+import sys
+import sqlite3
+
 class CustomerStatusManager():
-	'''Method: changes user status from inactive to active
-		Arguments: user, which indicates which user's status to change
-		Author: L.Sales, Python Ponies
-	'''
+    '''Method: changes user status from inactive to active
+        Arguments: user, which indicates which user's status to change. Active = 1, inactive = 0.
+        Author: L.Sales, Python Ponies
+    '''
+
+    def change_status(self, user):
+        with sqlite3.connect('../bangazon.db') as conn:
+            c = conn.cursor()
 
-	def change_status(user):
-		user.active = True;
\ No newline at end of file
+            try:
+                c.execute("UPDATE Customers SET active=0 WHERE active=1")
+                c.execute("UPDATE Customers SET active=1 WHERE customer_name = '{}'".format(user))
+            except sqlite3.OperationalError:
+                return "There was an error. Please try again."
+          
+if __name__ == '__main__':
+    activeUser = CustomerStatusManager()
+    activeUser.change_status("John Doe")
```

Comment:
We may need to refactor this line in the future to accommodate all users and not just John Doe.